examples/ola-recorder.cpp

```diff
@@ -115,36 +121,51 @@ int VerifyShow(const string &filename) {
     return ola::EXIT_NOINPUT;
 
   map<unsigned int, unsigned int> frames_by_universe;
-  uint64_t total_time = 0;
 
-  unsigned int universe;
-  ola::DmxBuffer buffer;
-  unsigned int timeout;
+  ShowEntry entry;
   ShowLoader::State state;
+  unsigned int playback_pos = 0;
+  bool playing = false;
   while (true) {
-    state = loader.NextFrame(&universe, &buffer);
+    state = loader.NextEntry(&entry);
     if (state != ShowLoader::OK)
       break;
-    frames_by_universe[universe]++;
-
-    state = loader.NextTimeout(&timeout);
-    if (state != ShowLoader::OK)
+    playback_pos += entry.next_wait;
+    if (FLAGS_stop > 0 && playback_pos >= FLAGS_stop) {
+      // Compensate for overshooting the stop time
+      playback_pos -= (playback_pos - FLAGS_stop);
       break;
-    total_time += timeout;
+    }
+    if (!playing && playback_pos >= FLAGS_start) {
+      // Found the start point
+      playing = true;
+      entry.next_wait = playback_pos - FLAGS_start;
```

Comment:
Isn't this irrelevant here? Or otherwise you're double subtracting FLAGS_start if it is used.